### ISSUE TITLE
Hardcoded values preventing design modification using only .Xib with cell 

### ISSUE THREAD

**Issue by krypt-lynx** [xmartlabs/Eureka#1762]
https://github.com/xmartlabs/Eureka/blob/master/Source/Core/Cell.swift#L127
This string overrides font color which was assigned in .Xib file before.

Also, looks like there is more of that  ".gray : .black" thing:

![image](https://user-images.githubusercontent.com/2183611/51631849-7a3d5880-1f56-11e9-9889-dbf853bcdd55.png)

**Comment by krypt-lynx** [comment]
by the way, it should be .darkText, at least

**Comment by mats-claassen** [comment]
You should set the colors for labels and text fields in `cellUpdate`. If you want to define it for all rows of one type then use `YourRow.defaultCellUpdate`

**Comment by mats-claassen** [comment]
I agree that it could be better but we want to handle the enabled/disabled state of the row, that is why the textColor is being set in code.
